.github/workflows/auto-set-base.yml

```diff
@@ -16,7 +16,18 @@ jobs:
       - name: Checkout Repository
         uses: actions/checkout@v3
 
+      - name: Check for "release-candidate" label
+        id: check-release-candidate
+        uses: actions/github-script@v6
+        with:
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+          script: |
+            const pr = context.payload.pull_request;
```

Comment:
Does GH run javascript code?  If so, TIL :-)

Reply:
yup, we do this in JS SDK for some labels, too 🙏

Reply:
Are we going to call these branches `rc-*` if so we could probably skip the label and this script and just edit the bash conditional to include that

Reply:
sure thing, I can do it that way, too. I updated it